CHANGELOG.md

```diff
@@ -5,19 +5,32 @@ Changelog for NeoFS Node
 
 ### Added
 
-- Retrieve passwords for storage wallets from the configuration in neofs-adm (#1539)
-
 ### Changed
 
 ### Fixed
 
+### Removed
+
+### Updated
+
+## [0.29.0] - 2022-07-07
+
+Support WalletConnect signature scheme.
+
+### Added
```

Comment:
- neofs-adm generates wallets in a pretty JSON format
- Makefile supports building from sources without a git repo

Reply:
Added